deploy/examples/onboarding-example.yaml

```diff
@@ -57,15 +57,15 @@ spec:
       uri: https://github.com/kiegroup/kogito-examples
       contextDir: onboarding-example/payroll
     imageRuntime:
-      imageStreamTag: 0.5.0
+      imageStreamTag: 0.6.0
     imageS2I:
-      imageStreamTag: 0.5.0
+      imageStreamTag: 0.6.0
     env:
       # optional, but will improve your build time quite a lot
       - name: MAVEN_MIRROR_URL
         value: "<http://yourmavenurl>"
   service:
     labels:
-      taxRate: process
-      vacationDays: process
-      paymentDate: process
+      taxes/rate: process
```

Comment:
Don't need to update on operator since they are dynamic, according to the service. I updated the examples: https://github.com/kiegroup/kogito-examples/pull/105